Typography/src/disclaimer/Disclaimer.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { TypographyProps } from '../base-typography/BaseTypography';
+import { disclaimerStyles, StyledDisclaimer } from './Disclaimer.styles';
+
+const Disclaimer = ({ children }: DisclaimerProps) => (
+  <StyledDisclaimer className={disclaimerStyles()}>{children}</StyledDisclaimer>
```

Comment:
Not sure where we landed on this, since we aren't passing any variants to this could we avoid the classname?

Reply:
This is likely me not fully groking the pattern yet. I was thinking i needed to call disclaimer styles all the time but I think what you're saying is in this example I could move those styles to StyledDisclaimer? If so, on the face of it, i don't like the inconsistency of where the styles go and the use of className. If className is how i apply styles, then it's weird to have styles from somewhere else?

Reply:
I've made the change but I'm torn. I dont like the extra noise of `className={disclaimerStyles()}` but i think i dislike having two different patterns more. Everyone has to learn when to do one or the other. I think we'll also wind up with a mix where people define the base styles in the styled component and the variants in the css function, especially if the components starts life without variants and then has variants added later.

547dc1c